app/code/Magento/Backend/Helper/Dashboard/Order.php

```diff
@@ -18,62 +16,52 @@ class Order extends \Magento\Backend\Helper\Dashboard\AbstractDashboard
     /**
      * @var \Magento\Reports\Model\ResourceModel\Order\Collection
      */
-    protected $_orderCollection;
+    private $orderCollection;
 
     /**
      * @var \Magento\Store\Model\StoreManagerInterface
      * @since 100.0.6
      */
-    protected $_storeManager;
+    private $storeManager;
 
     /**
      * @param \Magento\Framework\App\Helper\Context $context
      * @param \Magento\Reports\Model\ResourceModel\Order\Collection $orderCollection
      */
     public function __construct(
         \Magento\Framework\App\Helper\Context $context,
-        \Magento\Reports\Model\ResourceModel\Order\Collection $orderCollection
+        \Magento\Reports\Model\ResourceModel\Order\Collection $orderCollection,
+        \Magento\Store\Model\StoreManagerInterface $storeManager
```

Comment:
Please make new constructor argument optional and as retrieve it by fallback from object manager.
More details: https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development